connect/src/com/telenor/connect/ConnectRefreshTokenMissingException.java

```diff
@@ -0,0 +1,19 @@
+package com.telenor.connect;
+
+public class ConnectRefreshTokenMissingException extends IllegalStateException {
```

Comment:
Hmm, bit unsure here. If the user is signed in but we don't have (refresh) tokens, then something is very wrong with the state, I'd say.